test/common/index.mjs

```diff
@@ -1,12 +1,7 @@
 // Flags: --experimental-modules
 /* eslint-disable node-core/require-common-first, node-core/required-modules */
 
-import { createRequireFromPath } from 'module';
-import { fileURLToPath as toPath } from 'url';
-
-function createRequire(metaUrl) {
-  return createRequireFromPath(toPath(metaUrl));
-}
+import { createRequire } from 'module';
```

Comment:
This gets rid of the confusing deprecation warning whenever a test runs that uses `common/index.mjs`.

Reply:
Have been meaning to do this for a while!